project_manager.py

```diff
@@ -8,6 +8,16 @@
 
 from .json_file import JsonFile
 
+SETTINGS_FILENAME = 'project_manager.sublime-settings'
+resolved_computer_name = ''
```

Comment:
Resolving the computer name was also a relatively expensive operation so cached it since it's not likely to change.